pkg/web/serviceInstancesHandler.go

```diff
@@ -20,10 +20,20 @@ func NewMobileServiceInstancesHandler(serviceInstanceLister mobile.ServiceInstan
 }
 
 func (msih *MobileServiceInstancesHandler) List(c echo.Context) error {
-	si, err := msih.serviceInstanceLister.List(msih.namespace)
+	si, err := msih.serviceInstanceLister.List()
 	if err != nil {
 		c.Logger().Errorf("error listing service instances %v", err)
 		return c.NoContent(http.StatusInternalServerError)
 	}
 	return c.JSON(http.StatusOK, si)
 }
+
+func (msih *MobileServiceInstancesHandler) Watch(c echo.Context) error {
+	watchInterface, err := msih.serviceInstanceLister.Watch()
+	if err != nil {
+		return err
+	}
+
+	ServeWS(c, watchInterface)
```

Comment:
I think `ServeWS` should return an error and it should returned to the client if there is a problem to establish the connection.